src/app/register/register.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit } from '@angular/core';
+import { RegisterService } from '../services/register/register.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.scss']
+})
+export class RegisterComponent implements OnInit {
+
+  username: string;
+  password: string;
+  confirmPsw: string;
+  usernameError: string;
+  passwordError: string;
+  confirmPswError: string;
+  messageSuccess: string;
+  messageFail: string;
+
+  constructor(private registerService: RegisterService, private router: Router) { }
+
+  ngOnInit() {
+  }
+
+  CheckUserName(username: string): boolean {
+    if (username !== undefined && username.length > 0) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  // checkUserExists(username: string): boolean {
```

Comment:
This was forgotten and needs to be re-activated or removed

Reply:
solved, thanks. I implemented it into the register-onclick event